lisp/ui/widgets/dbmeter.py

```diff
@@ -77,6 +87,35 @@ def plot(self, peaks, _, decayPeak):
 
         self.update()
 
+    def updateMarkings(self):
+        self._markings = []
+
+        height = self.height()
+        font_height = QFontMetrics(self.font()).height()
```

Comment:
To optimize the space we can "remove" some empty space on top of the numbers, using `QFontMetrics.descent()` should provide the size of this value

```suggestion
        fm = QFontMetrics(self.font())
        font_height = fm.height() - fm.descent()
```

Reply:
According to the spec:
> `QFontMetrics::height()` [...] is always equal to `ascent()`+`descent()`.

So, we can use `fm.ascent()`, and save ourselves a calculation.